scripts/maverick.js

```diff
@@ -0,0 +1,7 @@
+function myDetails(name, id ,email,language, stage) {
```

Comment:
It's not wrong but I would suggest that you remove stage as a parameter of the myDetails function and rather hard code the stage value within the return string itself.
```suggestion
function myDetails(name, id ,email,language) {
    return `Hello World, this is ${name} with HNGi7 ID ${id} and email ${email} using ${language} for stage 2 task`
};
```

Reply:
Check your result and submit new PR is fail